bot/cogs/help_channels.py

```diff
@@ -223,7 +226,7 @@ async def close_command(self, ctx: commands.Context) -> None:
         if ctx.channel.category == self.in_use_category:
             if await self.dormant_check(ctx):
                 with suppress(KeyError):
-                    del self.help_channel_claimants[ctx.channel]
+                    await self.help_channel_claimants.delete(ctx.channel.id)
```

Comment:
I think instead suppressing this should use try-except with some logging.

Reply:
I disagree. We want to delete a mapping (channel -> claimant) that may not exist, so we either delete (goal accomplished) or we observe that it did not exist in the first place (goal accomplished). The latter situation is less likely now that we're using caching, but we're okay with it happening here. The goal is to not have the mapping anyway.

Reply:
I agree that this is a nice use case for `suppress`, I do not see the need to log whether the key existed or not since it doesn't affect any logic further down the line.

Reply:
Correct me if I'm wrong, but why are we using suppress here? If a key that does not exist is requested to be deleted in Redis it should not raise a KeyError, in the RedisCache docs that @lemonsaurus wrote it states that deleting non-existent keys will be silent.

Reply:
yeah @j03b is right. it's not necessary.

Reply:
Addressed in https://github.com/python-discord/bot/pull/987/commits/15dbbcf865dd24f5f8697fd85bd60d53d9450fcf